lib/parser/parsing.h

```diff
@@ -50,7 +50,8 @@ class Parsing {
 
   bool consumedWholeFile() const { return consumedWholeFile_; }
   const char *finalRestingPlace() const { return finalRestingPlace_; }
-  CookedSource &cooked() { return cooked_; }
+  CookedSource &cooked() { return *cooked_; }
+  std::unique_ptr<CookedSource> MoveCooked() { return std::move(cooked_); }
```

Comment:
I'm looking for ideas for a better name for this function.

Reply:
Transfer or Relinquish?